toxcore/tox.api.h

```diff
@@ -2135,6 +2135,17 @@ namespace conference {
   }
 
 
+  /**
+   * This event is triggered when the client successfully connects to a conference.
```

Comment:
Can you document somewhere what guarantees this gives the client? Perhaps in the docs for "invite", explain in what state it's valid to call "invite", and when it stops being valid.